### PR TITLE
Fix for #18 along with other tweaks.

### PR DESCRIPTION
**Pull request by waylaidwanderer**
I would also like to note that in the original material design, individual elements have their own special ripple properties, e.g. the blue button receives `background-color: #2a56c6` as well, not just having an opacity applied to it. This could perhaps be something to be worked on in the future.

### CODE REVIEW

File: scripts/ripples.js [hunk 1]
```diff
@@ -37,6 +37,9 @@ var ripples = {
                 rippleEnd       = new CustomEvent("rippleEnd", {detail: $ripple}),
                 refreshElementStyle;
 
+            // remove old ripples
+            $($rippleWrapper).find(".ripple").remove();
```
Comment (FezVrasta) [inline scripts/ripples.js]: as noted before, ripples.js **must** allow multiple ripples on the same element, if you remove the old ones only one ripple will be visible at time.

**Comment by waylaidwanderer** [comment]
I fixed it for real this time, I promise :P

**Comment by FezVrasta** [comment]
thanks, just one problem. ripples.js has not dependencies (no jQuery), so it should not use jquery functions.

I'll test your PR and see if it works correctly, if so we can think about write it in pure javascript

**Comment by waylaidwanderer** [comment]
> this make the ripple disappear even if the mouse is held down if there is only one ripple. Unfortunately this is not a solution :(

Are you sure? I specifically tested that and it worked as expected.

**Comment by waylaidwanderer** [comment]
Example: spam-click an element a bunch of times, and keep the mouse pressed down. Let go of the mouse, and you can see that the element becomes lighter (because the ripple fades out). That should be the expected behaviour which I just tested in Firefox.

**Comment by waylaidwanderer** [comment]
> thanks, just one problem. ripples.js has not dependencies (no jQuery), so it should not use jquery functions.

By the way, aren't `$` variables jQuery? If you truly wanted it to be not dependent, shouldn't you be using `var` to declare variables instead?

**Comment by waylaidwanderer** [comment]
Just tested in Chrome as well, and the ripples work as intended.

**Comment by FezVrasta** [comment]
1. Yes but it makes the other ripples fade out as well... I can see that probably is not easily noticeable.
2. I use the $ prefix to specify that the variable refers to a DOM object.

**Comment by waylaidwanderer** [comment]
This is really an edge case, as the user won't usually be spamming mouse clicks on an element, but I believe this works well enough for now. If you're willing to accept it as a pull request, I will figure out how to do it in javascript instead of jQuery and update the pull request to change that along with updating the fork to your latest changes.

**Comment by waylaidwanderer** [comment]
What do you think?

**Comment by FezVrasta** [comment]
I had no time to test this PR, will do some test asap

**Comment by FezVrasta** [comment]
Hey thanks! I've followed your idea and modified a bit, now seems to work very well. I've put credits to you in the commit.

5b249397d5d82ae88ab53b424881d50ccdd0bc03

**Pull request closed without merging**